tests/simulator/test_dict_serializable_restarts.py

```
@@ -0,0 +1,284 @@
+simArgs = dict(
+    time_step_nbr=1,
+    time_step=timestep,
+    boundary_types="periodic",
+    cells=200,
+    dl=0.3,
+    diag_options=dict(format="phareh5", options=dict(dir=out, mode="overwrite")),
+    restart_options=dict(dir=out, mode="overwrite"),
+)
+
+
+def dup(dic={}):
```

Ruff (0.8.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)